service-base/src/test/java/org/eclipse/hono/service/credentials/BaseCredentialsServiceTest.java

```diff
@@ -11,142 +11,176 @@
  */
 package org.eclipse.hono.service.credentials;
 
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
-import static org.eclipse.hono.util.CredentialsConstants.FIELD_SECRETS;
+import static java.net.HttpURLConnection.HTTP_CREATED;
+import static java.net.HttpURLConnection.HTTP_BAD_REQUEST;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
 
 import org.eclipse.hono.config.ServiceConfigProperties;
+import org.eclipse.hono.util.CredentialsConstants;
 import org.eclipse.hono.util.CredentialsResult;
+import org.eclipse.hono.util.MessageHelper;
 import org.junit.Before;
 import org.junit.Test;
 
 import io.vertx.core.AsyncResult;
+import io.vertx.core.Future;
 import io.vertx.core.Handler;
+import io.vertx.core.eventbus.Message;
 import io.vertx.core.json.JsonArray;
 import io.vertx.core.json.JsonObject;
 
 /**
  * Unit tests for BaseCredentialsService
- *
  */
 public class BaseCredentialsServiceTest {
 
     private BaseCredentialsService<ServiceConfigProperties> service;
 
+    private static final String OPERATION_ADD = "add";
```

Comment:
CredentialsConstants already defines all of the following constants, so please use them instead of redefining own own ones here. Valid for several lines...